src/rankcorr.jl

```diff
@@ -110,7 +110,7 @@ corkendall(X::RealMatrix, Y::RealMatrix) = Float64[corkendall!(float(X[:,i]), fl
 
 function corkendall(X::RealMatrix)
     n = size(X, 2)
-    C = eye(n)
+    C = Matrix(Diagonal(ones(n)))
```

Comment:
Isn't there a more efficient way to do this using `I`? IIUC this will allocate the `ones(n)` vector and discard it immediately.

Reply:
There is in 0.7 but it's purposefully not in Compat due to the change in default element type for `I`. Not sure what else would be good. I guess I could do something like
```julia
C = zeros(Float64, n, n)
C[diagind(C)] .= 1.0
```
if that would be better, or just live with the `eye` deprecation...

Reply:
OK. Maybe just add a version check? That will allow Femtocleaner to retain only the new efficient and concise approach automatically.